syntax = "proto3";

service ComputationService {
    rpc Add (AddRequest) returns (AddResponse);
    rpc Sort (SortRequest) returns (SortResponse);
}

message AddRequest {
    int32 i = 1;
    int32 j = 2;
}

message AddResponse {
    int32 result = 1;
}

message SortRequest {
    repeated int32 array = 1;
}

message SortResponse {
    repeated int32 sorted_array = 1;
}
